### PR TITLE
Members bug fixed and constants modifies

### PR DESCRIPTION
**Pull request by NitinCopods**

### CODE REVIEW

File: app/components/results/CandidateListOfTest.tsx [hunk 1]
```diff
@@ -22,22 +22,29 @@ const CandidateListOfTest = () => {
       route: 'completed',
     },
   ]
+  let navigate = useNavigate();
   const { testPreview, params } = useLoaderData()
   return (
     <div id="test-details" className=" h-full ">
```
Comment (arpita-20) [inline app/components/results/CandidateListOfTest.tsx:28]: remove extra space from className='h-full'

File: app/components/tests/TestList.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ const TestList = ({
   }, [sortDirection, sortBy])
   const showCheckBox = false
   return (
-    <div className="test-list-container flex h-full flex-col gap-6">
+    <div className="test-list-container flex h-full flex-col gap-12 p-1">
```
Comment (arpita-20) [inline app/components/tests/TestList.tsx:49]: remove p-1 as it is not required according to figma

Comment (arpita-20) [inline app/components/tests/TestList.tsx:49]: also remove gap-12.. add gap-10 according to figma

File: app/components/tests/TestTableItem.tsx [hunk 3]
```diff
@@ -33,7 +32,8 @@ const TestTableItem = ({
   status: string | undefined
 }) => {
   const [showDeletePopup, setShowDeletePopup] = useState(false)
-  const submit = useSubmit()
+  const submit = useSubmit();
+  const navigate = useNavigate();
   const deleteTest = () => {
     submit(
```
Comment (arpita-20) [inline app/components/tests/TestTableItem.tsx:38]: add gap-3 in line number 53

File: app/components/tests/TestList.tsx (hunk 2 above)
Comment (NitinCopods) [reply app/components/tests/TestList.tsx:49]: Outline is not coming properly, and there is nothing to i can do to fix it, that's why used padding and also i have talked to anurag regarding this

**Review by anuragP04 — APPROVED** [review]

**Pull request merged**